docs/workloads/gcp/wordpress-gcp.md

```diff
@@ -74,7 +74,7 @@ For the next steps, make sure your `kubectl` context points to the cluster where
   The `Application Developer(s)` will use the `complex` namespace.
 
   ```bash
-  kubectl apply -f cluster/examples/workloads/kubernetes/wordpress-gcp/cluster.yaml
+  kubectl apply -f cluster/examples/workloads/kubernetes/cluster.yaml
```

Comment:
I like this movement toward a more "portable" pattern here. Once strongly-typed resource classes are implemented we can add policies and make this truly portable. One thing that might be useful here as you bring the common AWS / GCP yaml to top-level would be to create a `wordpress` folder. I am anticipating further examples in this `kubernetes` directory, so it may be nice to have the following:
```
cluster/examples/workloads/kubernetes/wordpress/
|_____ cluster.yaml
|_____ app.yaml
|_____ aws/
       |_____ provider.yaml
|_____ gcp/
       |_____ provider.yaml
```
Note: this would require an update in line 39 in this file and line 151 in the corresponding AWS doc.

Reply:
I was thinking that too. My only worry is that the change becomes too radical. I just go ahead and do that